freshmaker/models.py

```diff
@@ -626,6 +626,23 @@ def get_lowest_build_id(cls, session):
             return 0
         return build.build_id
 
+    @classmethod
+    def get_most_original_nvr(cls, nvr):
```

Comment:
Would be nice to have unit test for this method too.

Reply:
Added.